integration/map-with-enums/map-with-enums.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package mapWithEnums;
+
+enum Enums {
+  UNKNOWN = 0;
+  GOOD = 1;
+  BAD = 2;
+}
+
+message MapWithEnums {
+  map<int32, Enums> enums_by_id = 1;
```

Comment:
Given this is such a simple scenario, can you edit `integration/simple/simple.proto` and add a `mapOfEnums` example to the `SimpleWithMap` message? That way this will live next to the other "map of ..." tests.

(And then similarly move your test into `simple-test.ts`.

Normally I like smaller repro tests like you did, but since each new `integration/...foo...` directory requires another codgen run / another pbjs run / etc., I've been nudging towards sharing `*.proto` files more often than not.

Sound okay?

Reply:
Sounds good, but this PR still has some more work coming. Map-with-enums.ts did not compile before the latest round of changes, and now that this proto has moved to simple, simple.ts no longer compiles. I'm working on figuring out what's up.